berserker-cassandra/src/main/java/io/smartcat/berserker/cassandra/worker/CassandraWorker.java

```diff
@@ -0,0 +1,133 @@
+import com.datastax.driver.core.Statement;
+
+import io.smartcat.berserker.api.Worker;
+import io.smartcat.berserker.cassandra.configuration.PreparedStatement;
+
+/**
+ * Worker that executes CQL statements on provided Cassandra connection points. It uses DataStax's java driver
+ * internally.
+ */
+public class CassandraWorker implements Worker<Map<String, Object>> {
+
+    private static final String QUERY = "query";
+    private static final String VALUES = "values";
+    private static final String PREPARED_STATEMENT_ID = "preparedStatementId";
+    private static final String CONSISTENCY_LEVEL = "consistencyLevel";
+
+    private final Cluster cluster;
+    private final Session session;
+    private final boolean async;
+    private final Map<String, com.datastax.driver.core.PreparedStatement> preparedStatements;
+
+    /**
+     * Constructs Cassandra worker with specified properties.
+     *
+     * @param connectionPointsWithPorts List of connection points to use.
+     * @param keyspace Name of keyspace in database to use.
+     * @param async Indicates whether statements will be executed synchronously or asynchronously.
+     * @param bootstrapDDLCommands List of CQL commands to execute only once after connection to Cassandra cluster is
+     *            established. Suitable for creating keyspaces, tables and populating some initial data if needed.
+     * @param prepStatements List of prepared statements to create. Each statement is defined with id and can be
+     *            referenced from {@link #accept(Map)} method.
+     */
+    public CassandraWorker(List<InetSocketAddress> connectionPointsWithPorts, String keyspace, boolean async,
+            List<String> bootstrapDDLCommands, List<PreparedStatement> prepStatements) {
+        if (connectionPointsWithPorts == null || connectionPointsWithPorts.isEmpty()) {
+            throw new IllegalArgumentException("List of connection points with ports cannot be null nor empty");
+        }
+        if (keyspace == null || keyspace.isEmpty()) {
+            throw new IllegalArgumentException("Keyspace cannot be null nor empty.");
+        }
+        cluster = Cluster.builder().addContactPointsWithPorts(connectionPointsWithPorts).build();
+        if (bootstrapDDLCommands == null || bootstrapDDLCommands.isEmpty()) {
+            session = cluster.connect(keyspace);
+        } else {
+            session = cluster.connect();
+            for (String command : bootstrapDDLCommands) {
+                session.execute(command);
+            }
+            session.execute("USE " + keyspace + ";");
+        }
+        preparedStatements = new HashMap<>();
+        if (prepStatements != null && !prepStatements.isEmpty()) {
+            for (PreparedStatement prepStatement : prepStatements) {
+                RegularStatement toPrepare = new SimpleStatement(prepStatement.getQuery());
+                com.datastax.driver.core.PreparedStatement prepared = session.prepare(toPrepare);
```

Comment:
You can remove intermediate variable so there is no need for the explicit package name
perparedStatements.put(id, session.prepare(toPrepare))